### DISCUSSION TITLE
Parse `package.json` funding section

### DISCUSSION THREAD

**Discussion by abitrolly** [community/community#5345] [Other Feature Feedback, Questions, & Ideas]
Projects like https://github.com/kach/nearley/ use `package.json` to signal the possible funding methods.

https://github.com/kach/nearley/blob/745c906fcc906a99c902a0ee652a8e5970e319eb/package.json#L59-L62

Would be good if GitHub could detect and use them.

**Comment by jlord** [comment]
Thanks, this is a great idea, we'd like to surface as much as we can who is sponsorable somewhere 👍
